gateway/config/config.go

```diff
@@ -9,11 +9,12 @@ type Config struct {
 
 // Project holds the project level configuration
 type Project struct {
-	Secret  string   `json:"secret" yaml:"secret"`
-	AESkey  string   `json:"aesKey" yaml:"aesKey"`
-	ID      string   `json:"id" yaml:"id"`
-	Name    string   `json:"name" yaml:"name"`
-	Modules *Modules `json:"modules" yaml:"modules"`
+	Secret      string   `json:"secret" yaml:"secret"`
+	AESkey      string   `json:"aesKey" yaml:"aesKey"`
+	ID          string   `json:"id" yaml:"id"`
+	Name        string   `json:"name" yaml:"name"`
+	Modules     *Modules `json:"modules" yaml:"modules"`
+	ContextTime int      `json:"contextTime" yaml:"contextTime"`
```

Comment:
@YourTechBud should we add comment to these individual fields ?
specifying what it does

Reply:
Just on the context time field since it isn't super clear